orbit/db.py

```diff
@@ -38,8 +38,7 @@ def insert_new(username: str, pwdhash: str, student_id: int):
         User.create(username=username, pwdhash=pwdhash, student_id=student_id)
 
     def set_pwdhash(username: str, pwdhash: str):
-        User.update(pwdhash=pwdhash).where(
-            Session.username == username).execute()
+        User.update(pwdhash=pwdhash).where(User.username == username).execute()
```

Comment:
the fact that the number of lines changes obfuscates what is actually going on here, but anyways should be melded into the commit that made the typo